main_zainu.ipynb

```diff
@@ -0,0 +1,137 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\"\"\"Importing the random module\"\"\"\n",
+    "import random"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Valid Input!! Can proceed\n"
+     ]
+    }
+   ],
+   "source": [
+    "\"\"\" A user-defined function to validate the input\n",
+    "If the input is a string, it parses the check\n",
+    "Else it print an error message\"\"\"\n",
+    "\n",
+    "def input_valid(input_string):\n",
+    "    if input_string.isdigit():\n",
```

Comment:
Nice work with the isdigit() method, have a think about what other checks you can put in!